python_modules/dagster/dagster/_utils/schedules.py

```diff
@@ -915,3 +915,17 @@ def get_latest_completed_cron_tick(
         execution_timezone=timezone,
     )
     return pendulum.instance(next(cron_iter))
+
+
+def get_next_cron_tick(
+    cron_string: Optional[str], current_time: datetime.datetime, timezone: Optional[str]
+) -> Optional[datetime.datetime]:
+    if not cron_string:
```

Comment:
should this fn just not take optional cron strings?

Reply:
agreed.